app/controllers/api/v1/greetings_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::GreetingsController < ApplicationController
+  def index
+    @greeting = Greeting.all.order('random()').first
+    render json: { greeting: @greeting.message }
+  end
```

Comment:
- Kindly make sure that you are correctly getting data from your database and that the URL is working as expected. as you see in the picture below you get this error `message is undefined `kindly fix that and make sure the app working as expected before the next submission. :+1: 
![image](https://user-images.githubusercontent.com/95297251/228707159-fe40edea-0b2e-4aa1-aedc-165e919fa3ab.png)

Reply:
Good evening

Thanks for your suggestion
This error message appears when Greetings TABLE is empty.

I've fixed the 'greetings_controller' to avoid this error message and display the message 'THERE ARE NOT GREETINGS ON THE API. PLEASE ADD THEM FROM BACKEND'

You can type `rails db:seed` from the backend terminal to automatically add 05 messages to the Greetings TABLE